pkg/k8s/init.go

```diff
@@ -43,7 +42,7 @@ func Init() error {
 
 		node := ParseNode(k8sNode)
 
-		log.Infof("Retrieved node's %s information from kubernetes", node.Name)
+		log.WithField(logfields.NodeName, node.Name).Info("Retrieved node's information from kubernetes")
```

Comment:
`"Received own node information from API server"`

Would be great to also print the Node IPs received as fields.